.github/workflows/create-release-pr.yml

```diff
@@ -27,25 +26,18 @@ jobs:
         id: set-release
         run: |
           if [[ $RELEASE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-            if [[ $PRE_RELEASE_VERSION =~ ^[a-z.0-9]+$ ]]; then
-              echo "release-tag: $RELEASE_VERSION$PRE_RELEASE_VERSION"
-              echo "release-tag=$RELEASE_VERSION$PRE_RELEASE_VERSION" >> $GITHUB_OUTPUT
-            elif [[ -n $PRE_RELEASE_VERSION ]]; then
-              echo "Input pre_release_version is not empty, but does not match the regex pattern ^[a-z.0-9]+$"
-              exit 1
-            else
-              echo "release-tag: $RELEASE_VERSION"
-              echo "release-tag=$RELEASE_VERSION" >> $GITHUB_OUTPUT
-            fi
+            echo "release-tag: $RELEASE_VERSION"
+            echo "release-tag=$RELEASE_VERSION" >> $GITHUB_OUTPUT
           else
-            echo "Version input doesn't match the regex pattern ^v[0-9]+\.[0-9]+\.[0-9]+$"
+            echo "Version input doesn't match the regex pattern ^[0-9]+\.[0-9]+\.[0-9]+$"
```

Comment:
Intentionally dropping the v moving forward? If so, update the if conditional above.